### PR TITLE
space fixes camera and new setting that changes camera behaviour

### PR DESCRIPTION
**Pull request by Defalt36**

### CODE REVIEW

**Comment by risvh** [comment]
may wanna include centerCamera.ini in the setting folder

**Comment by Defalt36** [comment]
Maybe in the future when I change the layout of the settings page I put the missing settings there, but I don't see a reason to.

**Comment by risvh** [comment]
Setting page doesn't include all the configurable settings, those omitted settings are configured with those ini files. Having the file there tells users what can be configured. If you place a setting in the code, better have it also in the setting folder for users' convenience.

File: gameSource/LivingLifePage.cpp [hunk 1]
```diff
@@ -17656,7 +17656,7 @@ void LivingLifePage::step() {
         doublePair screenTargetPos = 
             mult( targetObjectPos, CELL_D );
         
-        if( vogMode ) {
+        if( vogMode || SettingsManager::getIntSetting( "centerCamera", 0 ) ) {
```
Comment (connorhsm) [inline gameSource/LivingLifePage.cpp:17659]: As indicated by @risvh , please do commit this settings file.

**Review by connorhsm — CHANGES_REQUESTED** [review]
Otherwise looks good to me.

**Review by connorhsm — APPROVED** [review]

**Pull request merged**